src/app/zap-templates/zcl/data-model/chip/application-launcher-cluster.xml

```diff
@@ -49,6 +51,11 @@ limitations under the License.
     <item name="applicationId" type="CHAR_STRING"/>
   </struct>
 
+  <struct name="ApplicationLauncherResponse">
```

Comment:
I don't see this struct definition in the spec.  Where it it coming from?

It looks like the payload of "Launch Response Command", but that's not a spec-level struct; it's just a list of fields...

Reply:
I've needed a data structure to share between the clusters and example app code. I've removed this enum and I've created a application-launcher-server.h where I've declared this.